src/main/java/core/basesyntax/service/impl/FileServiceCsvImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileService;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileServiceCsvImpl implements FileService {
+    @Override
+    public List<String> readFile(String fileName) {
+        try {
+            return Files.readAllLines(Path.of(fileName));
+        } catch (FileNotFoundException e) {
```

Comment:
FileNotFoundException is a subclass of IOException, so catching FileNotFoundException is redundant here. It's enough to catch IOException which also covers FileNotFoundException.